controllers/v1/observationSubmissionsController.js

```diff
@@ -213,16 +213,15 @@ module.exports = class ObservationSubmissions extends Abstract {
           entityProfile: {},
           status: "started",
           scoringSystem: solutionDocument.scoringSystem,
-          isRubricDriven: solutionDocument.isRubricDriven
+          isRubricDriven: solutionDocument.isRubricDriven,
+          userRoleInformation:observationDocument.userRoleInformation
       };
 
       if( solutionDocument.hasOwnProperty("criteriaLevelReport") ) {
         submissionDocument["criteriaLevelReport"] = solutionDocument["criteriaLevelReport"];
       }
        
-      if (req.body && req.body.role) {
```

Comment:
@VISHNUDAS-tunerlabs should we not keep this check, basically if observation document(some records) don't have userRoleInformation then we can use what is being in the request by app ?